python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -0,0 +1,1219 @@
+    """
+    datasetType = pexConfig.Field(
+        dtype=str,
+        doc="Dataset type for input data; users will typically leave this alone, "
+        "but camera-specific ISR tasks will override it.",
+        default="raw",
+    )
+    expectWcs = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Expect input science images to have a WCS (set False for e.g. spectrographs)."
+    )
+    qa = pexConfig.ConfigField(
+        dtype=isrQa.IsrQaConfig,
+        doc="QA related configuration options.",
+    )
+    doHeaderProvenance = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Write calibration identifiers into output exposure header.",
+    )
+
+    # Differential non-linearity correction.
+    doDiffNonLinearCorrection = pexConfig.Field(
+        dtype=bool,
+        doc="Do differential non-linearity correction?",
+        default=True,
+    )
+
+    doOverscan = pexConfig.Field(
+        dtype=bool,
+        doc="Do overscan subtraction?",
+        default=True,
+    )
+    overscan = pexConfig.ConfigurableField(
+        target=OverscanCorrectionTask,
+        doc="Overscan subtraction task for image segments.",
+    )
+
+    # Amplifier to CCD assembly configuration.
+    doAssembleCcd = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Assemble amp-level exposures into a ccd-level exposure?"
+    )
+    assembleCcd = pexConfig.ConfigurableField(
+        target=AssembleCcdTask,
+        doc="CCD assembly task.",
+    )
+
+    # Combine snaps.
+    doSnapCombine = pexConfig.Field(
+        dtype=bool,
+        doc="Combine snaps?",
+        default=False,
+    )
+
+    # Bias subtraction.
+    doBias = pexConfig.Field(
+        dtype=bool,
+        doc="Apply bias frame correction?",
+        default=True,
+    )
+
+    # Deferred charge correction.
+    doDeferredCharge = pexConfig.Field(
+        dtype=bool,
+        doc="Apply deferred charge correction?",
+        default=False,
+    )
+    deferredChargeCorrection = pexConfig.ConfigurableField(
+        target=DeferredChargeTask,
+        doc="Deferred charge correction task.",
+    )
+
+    # Linearization.
+    doLinearize = pexConfig.Field(
+        dtype=bool,
+        doc="Correct for nonlinearity of the detector's response?",
+        default=True,
+    )
+
+    # Normalize gain.
+    doGainNormalize = pexConfig.Field(
+        dtype=bool,
+        doc="Normalize by the gain.",
+        default=True,
+    )
+
+    # Variance construction.
+    doVariance = pexConfig.Field(
+        dtype=bool,
+        doc="Calculate variance?",
+        default=True
+    )
+    gain = pexConfig.Field(
+        dtype=float,
+        doc="The gain to use if no Detector is present in the Exposure (ignored if NaN).",
+        default=float("NaN"),
+    )
+    usePtcGains = pexConfig.Field(
+        dtype=bool,
+        doc="Use the gain values from the Photon Transfer Curve?",
+        default=False,
```

Comment:
This should default to True, this is how we want the default run of the task to go.